sdk/build.gradle.kts

```diff
@@ -30,8 +30,7 @@ testModuleInfo {
     requires("org.junit.jupiter.api")
     requires("org.junit.jupiter.params")
     requires("org.mockito")
-
-    requiresStatic("java.annotation")
+    requires("java.annotation")
```

Comment:
```suggestion
```

Should be removed instead of changed.

Reply:
This is caused by update of dependency analysis plugin. Same as: https://github.com/hashgraph/hedera-services/pull/17876